src/sagemaker_inference/default_inference_handler.py

```diff
@@ -19,7 +19,8 @@ class DefaultInferenceHandler(object):
     """Bare-bones implementation of default inference functions.
     """
 
-    def default_model_fn(self, model_dir):
+    @staticmethod
+    def default_model_fn(model_dir):
```

Comment:
this doesn't break anything?

Reply:
These methods do not use `self` and do not rely on the state of the `DefaultInferenceHandler` object. Unit tests all passed. (If you have concerns that it might still break functionality, I can revert these changes and instead disable the `no-self-use` pylint check for these methods.)

Reply:
Reverted in https://github.com/aws/sagemaker-inference-toolkit/pull/28

Reply:
yeah, we don't have good tests around the toolkit actually being used in PyTorch/MXNet until it's too late, so I'd rather err on the safe side first